app/utils/types/raffle_types.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+class RaffleStatusType(str, Enum):
+    creation = "creation"  # Can edit every parameter
+    open = "open"  # Ordering is possible, no edition possible
+    locked = "locked"  # Can't order
```

Comment:
Maybe using more explicit label could be useful, maybe `editable`, `open`, `locked`
The comments may be more explicit